src/webgpu/api/validation/encoding/queries/begin_end.spec.ts

```diff
@@ -9,15 +9,50 @@ TODO:
     - begin 2, end 2
     - }
     - x= {
-        - render pass + occlusion
         - render pass + pipeline statistics
         - compute pass + pipeline statistics
         - }
-- nesting: test whether it's allowed to nest various types of queries
-  (including writeTimestamp inside begin/endable queries).
 `;
 
 import { makeTestGroup } from '../../../../../common/framework/test_group.js';
 import { ValidationTest } from '../../validation_test.js';
 
 export const g = makeTestGroup(ValidationTest);
+
+g.test('occlusion_query,begin_end')
+  .desc(
+    `
+Tests that begin/end occlusion query on render pass:
+- begin 0
+- begin 0, end (control case)
+- begin 0, begin 1, end
+- begin 0, end, end
+- end
+  `
+  )
+  .unimplemented();
+
+g.test('occlusion_query,begin_with_same_query_index')
+  .desc(
+    `
+Tests that begin occlusion query with same query index twice:
+- on {same, different (control case)} render pass
+  `
+  )
+  .unimplemented();
+
+g.test('nesting')
+  .desc(
+    `
+Tests that whether it's allowed to nest various types of queries:
```

Comment:
Either test occlusion-in-occlusion/pipelinestatistics-in-pipelinestatistics here, or add a note to the description saying they're tested elsewhere. IMO should just test them here, it's okay if there is duplicated test coverage.